style.css

```diff
@@ -77,18 +37,14 @@ form {
   width: 3rem;
 }
 
-/* #gif {
-  width: 100%;
-  text-align: center;
-} */
-
 #website {
   text-align: center;
   font-size: 35px;
-  margin: 0.75em;
-  position: relative;
-  top: 30%;
+  padding: 0.75em;
```

Comment:
There's a mix of em, rem, and px measurements for this page. Why not px exclusively?